### DISCUSSION TITLE
xrdp access login attempt limit?

### DISCUSSION THREAD

**Discussion by zinwelzl** [neutrinolabs/xrdp#3180] [Q&A] [answered]
Hi.

Can I setup, just with xrdp no others programs, access limit for 3 login attempt, and if that fail, user need to wait 10 minutes?

**Comment by rowlap** [comment, marked as answer]
You could try modifying the `xrdp-sesman` PAM configuration to make use of the `pam_faillock` module.

**Reply by matt335672** [reply]
Agreed - [pam_faillock(8)](https://www.man7.org/linux/man-pages/man8/pam_faillock.8.html) is exactly the way to go here. This functionality is not provided within the application itself - we rely on PAM to do what the system administrator has configured.
